### PR TITLE
Improve pull request template

### LINKED ISSUE #411: Pull request template enhancement
To remind contributors to update  `resources/connext_dds/CMakeList.txt` if they added a new `C/C++` example we can add something like that to the `PULL_REQUEST_TEMPLATE.md` to the master branch.

```markdown
<!-- Uncomment bellow if you added a C/C++ example and updated examples/connext_dds/CMakeList.txt 
- [ ] I added a new C/C++ example and updated `examples/connext_dds/CMakeList.txt`
-->
```

### PR DESCRIPTION
**Pull request by ManuelJNunez**
### Summary

I updated the `PULL_REQUEST_TEMPLATE.md` to remind everyone to update the top-level CMake when a new `C/C++` example is added.

### Details and comments

close #411

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,14 +2,19 @@
...
 ### Details and comments
 
+
+### Checks
+
+<!-- Change te space between the square brackets to an `x` -->
+-   [ ] I have updated the documentation accordingly.
+-   [ ] I have read the [CONTRIBUTING](https://github.com/rticommunity/rticonnextdds-examples/blob/develop/CONTRIBUTING.md) document.
+<!-- Uncomment bellow if you added a C/C++ example and updated examples/connext_dds/CMakeList.txt 
+-   [ ] I added a new C/C++ example and updated `resources/connext_dds/CMakeList.txt`.
```
Comment (lulivi) [inline .github/PULL_REQUEST_TEMPLATE.md:19]:
```suggestion
-   [ ] I added a new C/C++ example and updated `examples/connext_dds/CMakeList.txt`.
```

**Review by lulivi — CHANGES_REQUESTED** [review]

**Review by angelrti — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,14 +2,20 @@
...
-
 ### Details and comments
 
+
+### Checks
+
+<!-- Change te space between the square brackets to an `x` -->
+-   [ ] I have updated the documentation accordingly.
+-   [ ] I have read the [CONTRIBUTING](https://github.com/rticommunity/rticonnextdds-examples/blob/develop/CONTRIBUTING.md) document.
+<!-- Uncomment bellow if you added a C/C++ example and updated examples/connext_dds/CMakeList.txt -->
```
Comment (angelrti) [inline .github/PULL_REQUEST_TEMPLATE.md:18]:
```suggestion
<!-- Uncomment the bellow bullet if you have added a C/C++ example -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,14 +2,20 @@
...
+<!--
+-   [ ] I added a new C/C++ example and updated `examples/connext_dds/CMakeList.txt`.
```
Comment (angelrti) [inline .github/PULL_REQUEST_TEMPLATE.md:20]:
```suggestion
-   [ ] I have added a new C/C++ example and updated `examples/connext_dds/CMakeList.txt`.
```

**Review by angelrti — CHANGES_REQUESTED** [review]

**Review by lulivi — DISMISSED** [review]

**Comment by lulivi** [comment]
Maybe here we could add the new check once the Grade execution is enabled

**Review by lulivi — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -15,3 +15,12 @@
 <!-- Change te space between the square brackets to an `x` -->
 -   [ ] I have updated the documentation accordingly.
 -   [ ] I have read the [CONTRIBUTING](https://github.com/rticommunity/rticonnextdds-examples/blob/develop/CONTRIBUTING.md) document.
+
+<!-- Uncomment bellow if you added a C/C++ example and updated examples/connext_dds/CMakeList.txt -->
+<!--
+-   [ ] I added a new C/C++ example and updated `examples/connext_dds/CMakeList.txt`.
```
Comment (fgaranda) [inline .github/PULL_REQUEST_TEMPLATE.md:21]:
```suggestion
-   [ ] I have added a new C/C++ example and updated `examples/connext_dds/CMakeList.txt` accordingly.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -15,3 +15,12 @@
...
+-->
+<!-- Uncomment bellow if you added a Java example and updated examples/connext_dds/settings.gradle -->
+<!--
+-   [ ] I added a new Java example and updated `examples/connext_dds/settings.gradle`.
```
Comment (fgaranda) [inline .github/PULL_REQUEST_TEMPLATE.md:25]:
```suggestion
-   [ ] I have added a new Java example and updated `examples/connext_dds/settings.gradle` accordingly.
```

**Review by fgaranda — CHANGES_REQUESTED** [review]

**Review by fgaranda — APPROVED** [review]

**Review by angelrti — APPROVED** [review]

**Pull request merged**